src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -445,9 +445,13 @@ public void run() {
                     try {
                         serverTask.execute();
                     } catch (RuntimeException e) {
-                        // If a runtime exception occurred in the server task, log and rethrow
-                        error("An error occurred while starting the server: " + e.getMessage(), e);
-                        throw e;
+                        // If devStop is true server was stopped with Ctl-c, do not throw exception
+                        AtomicBoolean devStop = getDevStop();
+                        if (devStop.get() == false) {
```

Comment:
In the `addShutdownHook` method where it has
```
                cleanUpTempConfig();
                cleanUpServerEnv();
                setDevStop(true);
```
, we should move setDevStop to be first thing that happens, otherwise there may be some race condition here.

Reply:
Addressed with https://github.com/OpenLiberty/ci.common/pull/102/commits/647948f0a2d2638e8766a646768e7cfae15c641d

Reply:
I just realized it would have worked both ways since the shutdown hook doesn't stop the server until after the above mentioned part.  But the new way is neater.